src/types/groupRoleMapping.ts

```diff
@@ -0,0 +1,74 @@
+import Role, { ToznyAPIRole } from './role'
+
+type RolesByClient = Record<string, Role[]>
+
+/**
+ * An object representing the realm & client roles to which a particular realm group maps.
+ */
+class GroupRoleMapping {
```

Comment:
though not reflected in the github UI, the git history correctly reflects that this was moved from `roleMapping.js`